30_ComputeShaderPathTracer/main.cpp

```diff
@@ -34,7 +37,7 @@ class ComputeShaderPathtracer final : public examples::SimpleWindowedApplication
 		};
 
 		constexpr static inline uint32_t2 WindowDimensions = { 1280, 720 };
-		constexpr static inline uint32_t FramesInFlight = 5;
+		constexpr static inline uint32_t FramesInFlight = 1;
```

Comment:
ekhm.... please use frames in flight

Reply:
you can ask @Erfan-Ahmadi to proof read your code / give you pointers

Reply:
Done